### PR TITLE
Remove redundant obsolete brave/tracking-protection

### PR DESCRIPTION
**Pull request by bbondy**
Fix https://github.com/brave/brave-browser/issues/5389

It's been moved inside of the rust ad-block lib

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [x] Android
  - [x] iOS
  - [x] Linux
  - [x] macOS
  - [x] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [x] iOS
  - [x] Linux
  - [x] macOS
  - [x] Windows
- [x] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [x] Ran `git rebase master` (if needed).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [x] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [x] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [x] https://github.com/brave/brave-browser/wiki/QA-Guide

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: components/brave_new_tab_ui/containers/newTab/stats.tsx [hunk 1]
```diff
@@ -30,7 +26,7 @@ class Stats extends React.Component<Props, {}> {
   }
 
   get estimatedTimeSaved () {
-    const estimatedMillisecondsSaved = (this.adblockCount + this.trackedBlockersCount) * this.millisecondsPerItem || 0
+    const estimatedMillisecondsSaved = (this.adblockCount) * this.millisecondsPerItem || 0
```
Comment (emerick) [inline components/brave_new_tab_ui/containers/newTab/stats.tsx]: Total nitpick, but I think the parens are unnecessary now.

File: components/brave_new_tab_ui/stories/fakeLocale.ts [hunk 2]
```diff
@@ -4,8 +4,7 @@
 
 const locale: any = {
   // stats
-  trackersBlocked: 'Trackers Blocked',
-  adsBlocked: 'Ads Blocked',
+  adsBlocked: '“Ads and Trackers Blocked',
```
Comment (emerick) [inline components/brave_new_tab_ui/stories/fakeLocale.ts]: Is that a stray double quote character in that string?

Comment (bbondy) [reply components/brave_new_tab_ui/stories/fakeLocale.ts]: It is indeed. It's test data so non visible but it shouldn't be there. Good eye. Fixed.

File: components/brave_new_tab_ui/containers/newTab/stats.tsx (hunk 1 above)
Comment (bbondy) [reply components/brave_new_tab_ui/containers/newTab/stats.tsx]: 👍 fixed.

**Comment by bbondy** [comment]
rebased and pushed

**Pull request merged**